packages/suite/src/views/onboarding/steps/DeviceTutorial.tsx

```diff
@@ -75,23 +69,35 @@ export const DeviceTutorial = () => {
                 return (
                     <>
                         <Translation id="TR_TREZOR_DEVICE_TUTORIAL_DESCRIPTION" />
-                        {isActionAbortable && (
-                            <ActionButton
-                                onClick={() =>
-                                    TrezorConnect.cancel(intl.formatMessage(messages.TR_CANCELLED))
-                                }
-                            >
-                                <Translation id="TR_SKIP" />
-                            </ActionButton>
-                        )}
+                        <ButtonContainer>
+                            {isActionAbortable && (
+                                <Button
+                                    variant="tertiary"
+                                    size="tiny"
+                                    onClick={() =>
```

Comment:
extract out of render

Reply:
ee7032b27950ae4712990440f21d6ee7b156973e